ddp-workspace/projects/ddp-sdk/src/lib/components/activityForm/activityQuestion.component.ts

```diff
@@ -18,9 +18,11 @@ import { delay, filter, map, shareReplay, startWith, takeUntil, tap } from 'rxjs
                   [validationRequested]="validationRequested$ | async"
                   (valueChanged)="enteredValue$.next($event)">
           </ddp-activity-answer>
-          <div class="ddp-activity-validation" *ngIf="errorMessage$ | async as errorMsg">
-              <ddp-validation-message [message]="errorMsg">
-              </ddp-validation-message>
+          <div *ngIf="block.shown">
+            <div class="ddp-activity-validation" *ngIf="errorMessage$ | async as errorMsg">
+                <ddp-validation-message [message]="errorMsg">
+                </ddp-validation-message>
+            </div>
           </div>
```

Comment:
I gave you wrong information during PRTy time. The Angular syntax won't just let you parenthesize the existing ngIf expression (I think issue here is the creation of the errorMsg variable).
Suggestion below gets rid of new `<div>` element. ng-container add nothing to DOM, which I think is the desired behavior.
```suggestion
          <ng-container *ngIf="block.shown">
            <div class="ddp-activity-validation" *ngIf="errorMessage$ | async as errorMsg">
                <ddp-validation-message [message]="errorMsg">
                </ddp-validation-message>
            </div>
        </ng-container>
```

Reply:
Oops, just saw this now. I'll try this.

Reply:
Done. `ng-container` did the trick!